### ISSUE TITLE
New version 1.1.34 not showing in UI Latest server version

### ISSUE THREAD

**Issue by Raito00** [Ylianst/MeshCentral#6567]
Hello!

Web UI -> Check server version - not showing Latest 1.1.34 version

![image](https://github.com/user-attachments/assets/b4a868f0-03a4-4af2-a219-2f9c1396e513)

**Comment by si458** [comment]
Hello
Erm it's vanished from npm !?
The latest is 1.1.33?
Will speak to @Ylianst

**Comment by si458** [comment]
It's there.
It just not marked as the latest version for some reason?
The web ui checker uses the `latest` tag on npm which explains why ur still seeing 1.1.33
Awaiting @Ylianst reply

**Comment by si458** [comment]
ok this has been done deliberately! 
when we released 1.1.33, we replaced powershell commands with wmi functions,
however these broke any windows meshagents running less then 8.1/server 2016 !
so @Ylianst pinned the latest to 1.1.33 to make sure people didnt update and then lose access to there remote devices!
i have pushed fixes today which fixes the issues thankfully!
so will try get a new release out soon/asap!
